### PR TITLE
Community PR test pass 1/18/2021

### PR DESCRIPTION
**Pull request by ryalanms**
**Community PR Test Pass 1/11/2021**

These PRs will be merged if no regressions are found.

**Community PRs included in this test branch:**

Remove custom copy of PtrToStringChars #3117
Using array empty to replace create an empty array #2843

**Test Matrix**

| OS | Architecture |
| --- | --- |
| Windows 7 SP1	Enterprise	| x86 |	
| Windows 7 SP1	Enterprise	| x64 |	
| Windows Blue	Enterprise	| x86 |	
| Windows Server 2012 R2 SP1	| x64 |
| Win10 RS1                	Enterprise	| x86 |
| Win10 RS3	Enterprise	| x64 |
| Win10 RS4	Enterprise	| x86 |
| Windows Server 2019		| x64  |
| Win10 RS6	Enterprise	| X86 |
| Windows Server 2016	 | x64 |

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Generated/AvTraceMessages.cs
Comment (SamBent) [inline src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Generated/AvTraceMessages.cs:85]: This is a generated file (see comment at the top of the file).  You need to make these changes in the template as well, or they'll get overwritten the next time the template changes. 

The right way to do this change is:
a. Change the template file AvTraceMessages.tt
b. Re-run the generator to propagate the changes to the 3 AvTraceMessages.cs files (for Base, Core, and FW).
c. Check in all 4 changed files.
I think step (b) happens automatically at the start of a full build, but you should verify that.

**Review by SamBent — CHANGES_REQUESTED** [review]

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Generated/AvTraceMessages.cs
Comment (lindexi) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/Generated/AvTraceMessages.cs:85]: @SamBent Thank you and I fix it in https://github.com/dotnet/wpf/pull/2843

**Comment by ryalanms** [comment]
I will wait for Sam's sign-off before starting the test pass.  If any other PRs come in before then, we can add them to this test run.

**Comment by ryalanms** [comment]
I am still waiting on sign-offs to start the test pass.  There are only two PRs, so this will have to wait until Monday.  Thanks.

**Pull request closed without merging**
